### PR TITLE
Update status code for missing Loopring key

### PR DESCRIPTION
**Pull request by yabirgb**
The status code that were used before can be triggered if the account
is not a loopring account and results in an error not handled.

The thing with this code is that is not returned when the key is not
valid and simply says that the user is not found (the codes we were using)
but since this codes can be returned in other situations I believe is a nice idea
to leave just this code.

### CODE REVIEW

File: rotkehlchen/chain/ethereum/modules/l2/loopring.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def _api_query(
                     # This two codes can appear if there is an error with the
                     # key as tested here
                     # https://github.com/rotki/rotki/pull/2569#issuecomment-800151821
-                    if code and code in (100001, 101002):
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/modules/l2/loopring.py:259]: So from the docs this error code is returned for invalid api key.

If that is the case you will need to edit the comment also and edit the exception. It's no longer an APIKeyMismatch but an invalid api key. Also adjust error messages where it's caught.

**Comment by yabirgb** [comment]
Tested this with an account with the following situation:

- 2 addresses registered at loopring
- 2 addresses not registered at loopring

In the cases of:

- Valid APIKey

I get in the logs

```
[18/03/2021 13:06:06 CET] DEBUG rotkehlchen.chain.ethereum.modules.l2.loopring: Querying loopring https://api3.loopring.io/api/v3/account?owner=<address1>
[18/03/2021 13:06:06 CET] DEBUG rotkehlchen.chain.ethereum.modules.l2.loopring: Skipping loopring query of address <address1> because is not registered at loopring
[18/03/2021 13:06:06 CET] DEBUG rotkehlchen.chain.ethereum.modules.l2.loopring: Querying loopring https://api3.loopring.io/api/v3/user/balances?accountId=<address2 id>
[18/03/2021 13:06:06 CET] DEBUG rotkehlchen.externalapis.coingecko: Querying coingecko: https://api.coingecko.com/api/v3/simple/price/?ids=loopring&vs_currencies=usd
[18/03/2021 13:06:07 CET] DEBUG rotkehlchen.chain.ethereum.modules.l2.loopring: Querying loopring https://api3.loopring.io/api/v3/account?owner=<address3>
[18/03/2021 13:06:07 CET] DEBUG rotkehlchen.chain.ethereum.modules.l2.loopring: Skipping loopring query of address <address3> because is not registered at loopring
[18/03/2021 13:06:07 CET] DEBUG rotkehlchen.chain.ethereum.modules.l2.loopring: Querying loopring https://api3.loopring.io/api/v3/user/balances?accountId=<address4 id>
```

- Invalid APIKey

I get in the UI the following two notifications

```
Error querying loopring address <address2>. Verify that the api key for loopring is correct.
Error querying loopring address <address4>. Verify that the api key for loopring is correct.
```

As before it skips the two addresses that are not in loopring

File: rotkehlchen/chain/ethereum/modules/l2/loopring.py [hunk 2]
```diff
@@ -253,11 +257,14 @@ def _api_query(
...
                 if result_info:
                     code = result_info.get('code', None)
-                    # This two codes can appear if there is an error with the
-                    # key as tested here
-                    # https://github.com/rotki/rotki/pull/2569#issuecomment-800151821
-                    if code and code in (100001, 101002):
-                        raise LoopringAPIKeyMismatch()
+                    # From the documentation the code returned when the ApiKey
+                    # is not valid is 104002 https://docs3.loopring.io/en/?q=104002
+                    if code and code == 104002:
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/modules/l2/loopring.py:262]: improvement/nitpick:

Here you are checking that `code` is not `None` 2 times.

You can:

```python
    if code is not None:
        if code == 104002:
            raise LoopringInvalidApiKey()
        elif code == 101002:
            raise LoopringUserNotFound()
```
```

**Review by LefterisJP — APPROVED** [review]
Approving since it looks good. But leaving the nitpick fix to you. If you want to do here do it. If another PR merge this and do it in another one.

**Comment by yabirgb** [comment]
nitpick solved. I tested again and I didn't break anything. Test passing. Merging it

**Pull request merged**